lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -2113,39 +2123,81 @@ def test_approved_email_message(self):
         )
 
         self.assertIn(
-            "Your verification for course {course_name} and assessment {assessment} has been passed.".format(
-                course_name=self.course.display_name_with_default,
-                assessment=self.assessment
+            "We have successfully verified your identity for the {assessment} "
+            "assessment in the {course_name} course.".format(
+                assessment=self.assessment,
+                course_name=self.course.display_name_with_default
             ),
             body
         )
 
+        self.check_courseware_link_exists(body)
         self.assertIn("Re-verification Status", subject)
 
     def test_denied_email_message_with_valid_due_date_and_attempts_allowed(self):
 
-        __, body = _compose_message_reverification_email(
+        subject, body = _compose_message_reverification_email(
             self.course.id, self.user.id, "midterm", self.attempt, "denied", True
         )
 
         self.assertIn(
-            "Your verification for course {course_name} and assessment {assessment} has failed.".format(
+            "We could not verify your identity for the {assessment} assessment "
+            "in the {course_name} course. You have used "
+            "{used_attempts} out of {allowed_attempts} attempts to "
+            "verify your identity.".format(
                 course_name=self.course.display_name_with_default,
-                assessment=self.assessment
+                assessment=self.assessment,
+                used_attempts=1,
+                allowed_attempts=self.allowed_attempts
             ),
             body
         )
 
-        self.assertIn("Assessment closes on {due_date}".format(due_date=get_default_time_display(self.due_date)), body)
-        self.assertIn("Click on link below to re-verify", body)
+        reverify_link = "https://{}{}".format(
+            microsite.get_value('SITE_NAME', 'localhost'), self.re_verification_link
+        )
+
+        self.assertIn(
+            "To try to verify your identity again, select the following "
+            "{reverify_link}.".format(
+                reverify_link=reverify_link
+            ),
+            body
+        )
+
+        self.assertIn("Re-verification Status", subject)
+
+    def test_denied_email_message_with_valid_due_date_and_no_attempts_allowed(self):
+
+        VerificationStatus.add_verification_status(
+            checkpoint=self.check_point,
+            user=self.user,
+            status='submitted',
+            location_id=self.reverification_location
+        )
+
+        __, body = _compose_message_reverification_email(
```

Comment:
You might need to change the parameters being passed.
Reference: https://github.com/edx/edx-platform/pull/8231